src/main/java/com/akathist/maven/plugins/launch4j/Launch4jMojo.java

```diff
@@ -72,9 +72,11 @@
 )
 public class Launch4jMojo extends AbstractMojo {
 
-    private static final String LAUNCH4J_ARTIFACT_ID = "launch4j";
+    // intentionally non-static non-final, so they can be changed
+    // via reflection if someone absolutely positively needs to
+    private String LAUNCH4J_ARTIFACT_ID = "launch4j";
 
-    private static final String LAUNCH4J_GROUP_ID = "net.sf.launch4j";
+    private String LAUNCH4J_GROUP_ID = "net.sf.launch4j";
```

Comment:
Wouldn't be better to [parameters](https://maven.apache.org/guides/plugin/guide-java-plugin-development.html#parameters)?

Reply:
I may be a little sleep-deprived. It would certainly be less spaghettish. I can't think of anything right now when 'normal' users would need to change it, so not sure if there's a point in completely exposing it, but I guess there's no harm either.

Reply:
There is one - you ;-)